src/Inciter/Transporter.h

```diff
@@ -195,7 +202,7 @@ class Transporter : public CBase_Transporter {
     void rowcomplete();
 
     //! Reduction target summing total mesh volume
-    void vol( tk::real v );
+    void totalvol( tk::real v );
```

Comment:
That's not a bad idea. But note that where there is `redn_` so far in the code, it is not my choice, but because we need to call a Charm++ macro (which starts with `redn_`). If we do this rename, though, I feel should be done consistently everywhere, so in another PR.